extensions/_beacons/http_status.py

```diff
@@ -105,30 +105,36 @@ def beacon(config):
                 r = requests.get(url, timeout=sites_config['timeout'])
             else:
                 r = requests.get(url, timeout=30)
+        # r.status_code
         except requests.exceptions.RequestException as e:
-            log.debug("Request failed: ", e)
+            log.debug("Request failed: %s", e)
+        if r.status_code >= 500:
+            log.debug('Response from status endpoint was invalid: '
+                      '%s', r.status_code)
         for json_response_item in sites_config['json_response']:
-            service = json_response_item['path'].split(':')[0]
-            service_value = json_response_item['path'].split(':')[1]
+            if ':' in json_response_item['path']:
+                service = json_response_item['path'].split(':')[0]
+                service_value = json_response_item['path'].split(':')[1]
```

Comment:
I still think that using the utils method for traversing dictionary paths would be better since it allows for arbitrary depth of nesting, vs. the shallow nesting that is happening in our status endpoint. This works for the current case, but is less flexible overall.

Reply:
I looked at using `traverse_dict` in the `utils.data` module however I don't think that would work as the path itself is not a dict but a string with a colon ex. `certificate:status` so when I try passing in the value as the key for that function, it does not appear to work

Reply:
The idea was to use `salt.utils.traverse_dict` for going over the response from the status endpoint and not pillar. Made the change based on feedback.